documentation/Readme.md

```diff
@@ -0,0 +1,47 @@
+# Project Title
```

Comment:
You have not added full stops in some of the points. I request you to correct them.

Reply:
Added full stops to complete sentences. Bullet points don't require full stops.

Reply:
ok